content/en/dynamic_instrumentation/enabling/java.md

```diff
@@ -44,7 +45,7 @@ The Datadog Dynamic Instrumentation library is supported in JDK version 8 and ab
 {{% /tab %}}
 {{< /tabs >}}
 
-   **Note**: Dynamic Instrumentation is available in the `dd-java-agent.jar` library in versions 1.24.0 and later.
+   **Note**: Dynamic Instrumentation is available in the `dd-java-agent.jar` library in versions 1.34.0 and later.
```

Comment:
not sure we want to put the version higher just for symdb

Reply:
@EWachnowezki wants to push people to use newer versions: https://github.com/DataDog/documentation/pull/23635#discussion_r1636367247

I agree that pushing people to use newer versions is good. We're (unfortunately) not enforcing it at the backend level, so users currently using DI won't be affected by the docs change.

Reply:
+1 on Oskar 's comment - I personally am in favor of making this change, I agree we should drive folks to the path where they have SymDB working when they first try our product.